plugins/azure-devops/src/utils/getAnnotationValuesFromEntity.ts

```diff
@@ -60,6 +66,7 @@ export function getAnnotationValuesFromEntity(entity: Entity): {
       return {
         project: projectRepo.project,
         repo: projectRepo.repo,
+        readmePath: readmePath,
```

Comment:
Can we please include tests for these as well?

Reply:
Done